Automattic/woocommerce-payments
# Update PO success modal

Fixes #7579

#### Changes proposed in this Pull Request
- Implement new PO success modal design (Figma: VLWDmeuIXQ7XAK4flyqTFO-fi-182_17764)
- Install the `canvas-confetti` package and type definitions.
- Create `ConfettiAnimaiton` component. We were waiting for the new release of the `@automatic/components` package to use the existing component there. However, after testing, I found that there are some typing issues, our bundle will grow more than expected, and the component doesn't fit well with the proposed design. So it's better to use our own component, since it's quite small and simple.

#### Testing instructions
Confirm that the implemented design matches the one in Figma, ensuring it's properly formatted in mobile size.

As this doesn't affect the logic, the best way to test it is to set the following condition to true:
https://github.com/Automattic/woocommerce-payments/blob/5634b1789b4e74baa489d4c27f3779fe03d18183/client/overview/index.js#L213
And comment this line:
https://github.com/Automattic/woocommerce-payments/blob/5634b1789b4e74baa489d4c27f3779fe03d18183/client/overview/modal/progressive-onboarding-eligibility/index.tsx#L65



#### Screenshots
| Before | After | After Mobile |
|--------|--------|--------|
|<img width="720" alt="Screenshot 2024-01-10 at 12 36 54" src="https://github.com/Automattic/woocommerce-payments/assets/7670276/c529d604-84a9-4981-ad04-5399558e69ef">|<img width="729" alt="Screenshot 2024-01-19 at 12 06 06" src="https://github.com/Automattic/woocommerce-payments/assets/7670276/fb76c5f3-f43b-4d39-a190-f46aeb5aa9de">|<img width="445" alt="Screenshot 2024-01-19 at 12 06 41" src="https://github.com/Automattic/woocommerce-payments/assets/7670276/2a30e64d-ab91-46ba-9c18-2443f2aad6ba">|

**Confetti animation**

https://github.com/Automattic/woocommerce-payments/assets/7670276/520c671b-0b02-438f-8637-a9712fc88131


-------------------


**Post merge**
